src/formatDate.js

```diff
@@ -50,7 +50,48 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const firstSeperator = fromFormat.pop();
+  const secondSeperator = toFormat.pop();
```

Comment:
you change your arguments, and if, by some reason, we need to use them, they won't have their last elements 😭

Reply:
pushed these values before return